src/oscontainer.py

```diff
@@ -61,7 +87,7 @@ def oscontainer_extract(containers_storage, src, dest,
     podCmd.append(src)
 
     run_verbose(podCmd)
-    inspect = run_get_json(['podman', rootarg, 'inspect', src])[0]
+    inspect = run_get_json_retry(['podman', rootarg, 'inspect', src])[0]
```

Comment:
`podman inspect` is a purely local operation I believe, so I don't think it makes sense to retry.

Reply:
Right, now retrying skopeo only